jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -4,10 +4,8 @@ properties:
     type: object
     properties:
       date:
-        anyOf:
-          - $ref: http://stsci.edu/schemas/asdf/time/time-1.0.0
-          - type: string
```

Comment:
@nden or @hbushouse, anyone remember why Time objects were added to the core schema?  These cannot be serialized to a FITS keyword.

Reply:
Nope. That's a question for either Mike D. or Bernie. I guess they thought it would be an elegant way to do validation of the formatting for the date+time keywords, but never really worked as hoped.

Reply:
https://github.com/spacetelescope/jwst/commit/6621c6fba120b38b460c462f1c34281221e35ffd

Reply:
@jdavies Why not? The value of a `Time` object in `isot` format is the value of the FITS keyword.
But that's not something to be done before a release. I think it's fine to remove those from schema now.